### PR TITLE
ja: Make /docs/tasks/tools/install-kubectl/ follow v1.17 of the original text

### PR DESCRIPTION
**Pull request by ryokotmng**
Issue https://github.com/kubernetes/website/issues/21401

Appreciate your review!

### CODE REVIEW

File: content/ja/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -381,6 +383,27 @@ Bashにおけるkubectlの補完スクリプトは`kubectl completion bash`コ
...
+brew install bash
+```
+
+シェルをリロードし、希望するバージョンを使用していることを確認してください。
+
+```shell
+echo $BASH_VERSION $SHELL
+```
+
+Homebrewは通常、`/usr/local/bin/bash`フォルダ下にインストールします。
```
Comment (ryokotmng) [inline content/ja/docs/tasks/tools/install-kubectl.md:406]: I'm afraid it may not be so good wording here🙁

**Comment by ryokotmng** [comment]
Would be great if someone can review this at one's convenience🙏

**Comment by oke-py** [comment]
@ryokotmng You don't need to assign PR to yourself. Please assign only Issue.
For PR, you should assign one of reviewers following k8s-ci-robot suggestion.
https://github.com/kubernetes/website/pull/21405#issuecomment-636893709

**Comment by ryokotmng** [comment]
Understood! Thank you for the advise🙇‍♂️

File: content/ja/docs/tasks/tools/install-kubectl.md (hunk 1 above)
Comment (nasa9084) [reply content/ja/docs/tasks/tools/install-kubectl.md:406]:
```suggestion
Homebrewは通常、`/usr/local/bin/bash`にインストールします。
```

`/usr/local/bin/bash` is not a directory, it is bash itself

Comment (ryokotmng) [reply content/ja/docs/tasks/tools/install-kubectl.md:406]: 🙇‍♂️

**Comment by ryokotmng** [comment]
@nasa9084 Thanks for your review! I fixed the wording as you kindly advise. Would be great if you could go through at your convenience:)

File: content/ja/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -80,22 +80,24 @@ yum install -y kubectl
...
+```
+{{% /tab %}}
+{{% tab name="Homebrew" %}}
+Linuxで[Homebrew](https://docs.brew.sh/Homebrew-on-Linux)パッケージマネージャーを使用している場合は、kubectlを[インストール](https://docs.brew.sh/Homebrew-on-Linux#install)することが可能です。
+```shell
 brew install kubectl
 
 kubectl version
-{{< /tab >}}
-{{< /tabs >}}
```
Comment (oke-py) [inline content/ja/docs/tasks/tools/install-kubectl.md:98]: この行は必要そうです。

Comment (oke-py) [reply content/ja/docs/tasks/tools/install-kubectl.md:98]: `{{< /tabs >}}`の方です。

Comment (ryokotmng) [reply content/ja/docs/tasks/tools/install-kubectl.md:98]: あ、本当ですね...🙇‍♂️ありがとうございます。修正し、今一度見直しました。お手すきで再レビューお願いします。

**Comment by ryokotmng** [comment]
レビューありがとうございました！もうちょい注意しないとですね💦

**Pull request merged**